### DISCUSSION TITLE
梅林 阿里网盘问题

### DISCUSSION THREAD

**Discussion by EllenWu9** [messense/aliyundrive-webdav#434] [Q&A] [answered]
refresh token通过Chrome获取，之前一直都ok，获取后运行，无法通过infuse连接，且梅林网盘插件一会儿进程失败？啥情况？

**Comment by wuxiaoqiang12** [comment, marked as answer]
请问你解决了吗？我也有类似问题

**Reply by EllenWu9** [reply]
已经解决，Merlinclash更新至220520版本，DNS方案选择Redir-Host；

**Reply by wuxiaoqiang12** [reply]
是的，我也发现了是dns方案选择了fake ip的问题，已经解决了。
